cmd/earthly/secret_cmds.go

```diff
@@ -276,17 +285,55 @@ func (app *earthlyApp) fullSecretPath(path string) (string, error) {
 		return path, nil
 	}
 
-	if app.orgName == "" {
-		return "", fmt.Errorf("the --org flag is required")
+	orgName, projectName, isPersonal, err := app.getOrgAndProject(ctx, cloudClient)
+	if err != nil {
+		return "", err
 	}
-	if app.projectName == "" {
-		return "", fmt.Errorf("the --project flag is required")
+
+	// TODO this is a bit ugly, consider having getOrgAndProject return isPersonal bool
```

Comment:
is this comment still valid? I see `isPersonal` comes from a few lines up where `app.getOrgAndProject` is called.